jwst/pipeline/calwebb_image2.py

```diff
@@ -51,8 +50,12 @@ def process(self, input):
 
             # Save result
             suffix = 'cal'
+            if isinstance(input, datamodels.CubeModel):
```

Comment:
good catch. this bug is in spec2 also. No regression for ints?  will have to make one.